internal/cli/atlas/backup/compliancepolicy/compliancepolicy.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package compliancepolicy
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func baseCommand() *cobra.Command {
+	const use = "compliancepolicy"
+	cmd := &cobra.Command{
+		Use:     use,
+		Hidden:  true,
+		Aliases: []string{"compliance-policy", "bcp"},
```

Comment:
1. Do we use this format with dash anywhere? If not, I'd say it's better to remove it from the list of aliases
2. `atlas backups bcp` sounds redundant (atlas backups backup compliance policy), do we need this alias? Alsom `cp` might sound like `copy`

Reply:
I'll remove the aliases.